src/shared/rpc-contracts/app-rpcs/finance.errors.ts

```
@@ -0,0 +1,53 @@
+  RpcBadRequestError,
+  RpcForbiddenError,
+  RpcInternalServerError,
+  RpcUnauthorizedError,
+} from '../../errors/rpc-errors';
+
+export type ReceiptMediaError =
+  | ReceiptMediaBadRequestError
+  | ReceiptMediaInternalError
+  | ReceiptMediaServiceUnavailableError;
+
+export class FinanceReceiptNotFoundError extends Schema.TaggedError<FinanceReceiptNotFoundError>()(
```

ESLint

[error] Use `new` when creating an error.

(unicorn/throw-new-error)